frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template.tsx

```diff
@@ -165,4 +161,26 @@ type VirtualMachineTemplatesPageProps = {
   match: match<{ ns?: string }>;
 };
 
-export { VirtualMachineTemplatesPage };
+const VMTemplateLink: React.FC<VMTemplateLinkProps> = ({ template }) => {
```

Comment:
could you move this component to a separate file?

Reply:
Yes :peace_symbol: it will look nicer in it's own file :+1:

Reply:
done